src/jit/codegenarm64.cpp

```diff
@@ -416,13 +416,17 @@ void CodeGen::genSaveCalleeSavedRegistersHelp(regMaskTP   regsToSaveMask,
                                               int         lowestCalleeSavedOffset,
                                               int         spDelta)
 {
+    assert(spDelta <= 0);
     unsigned regsToSaveCount = genCountBits(regsToSaveMask);
     if (regsToSaveCount == 0)
     {
+        if (spDelta != 0)
+        {
+            genStackPointerAdjustment(spDelta, REG_NA, nullptr);
+        }
```

Comment:
It would be useful to add a comment to the genStackPointerAdjustment declaration indicating "tmpReg can be REG_NA when caller "knows" that the immediate will always fit in the instruction". Should there be an assert about the range of spDelta? Is it only going to be 0 or 8, for alignment?